content/tutorial-x-ray-image-processing.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<p>The <code>biomedical-images</code>  folder needs to be renamed here too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Good catch. Done! `/tutorial-x-ray-image-processing`